### PR TITLE
doc: exec no privileged option.

### PR DESCRIPTION
**Pull request by lloydde**
https://github.com/docker/docker/blob/master/docs/sources/reference/commandline/cli.md#exec
Also:
- textwidth=79

### CODE REVIEW

File: docs/api/commands/exec.md [hunk 1]
```diff
@@ -44,7 +45,9 @@ This will create a new Bash session in the container `ubuntu_bash`.
 
 ## Divergence
 
-There is no known divergence between the Triton SDC Docker and Docker Inc. implementations of this method. Please contact Joyent support or file a ticket if you discover any.
+There is no known divergence between the Triton SDC Docker and Docker Inc.
```
Comment (misterbisson) [inline docs/api/commands/exec.md]: This is not correct if `--privileged` is unsupported.

File: docs/api/commands/exec.md [hunk 2]
```diff
@@ -44,7 +45,9 @@ This will create a new Bash session in the container `ubuntu_bash`.
...
+implementations of this method. Please contact Joyent support or file a ticket
```
Comment (misterbisson) [inline docs/api/commands/exec.md]: Please don't hard-wrap english language content. It breaks the semantics of the language.

Comment (jclulow) [inline docs/api/commands/exec.md]: It's actually just fine to carry on hard-wrapping _Markdown Source_, which is what this file is.  The semantic construct of a paragraph, in Markdown, is a contiguous set of lines separated by a blank line.  This is essentially a code style thing, and is generally a per-repository decision.

File: docs/api/commands/exec.md (hunk 1 above)
Comment (lloydde) [inline docs/api/commands/exec.md]: Sorry, I'm a little understanding. "--privileged" has been removed from the upstream documentation for this command, see https://github.com/docker/docker/blob/master/docs/sources/reference/commandline/cli.md#exec

File: docs/api/commands/exec.md (hunk 2 above)
Comment (lloydde) [inline docs/api/commands/exec.md]: @misterbisson you want this one paragraph formatted differently than the rest of the file? I'd highly recommend against changing the format from upstream.

**Pull request closed without merging**